src/style.css

```diff
@@ -0,0 +1,90 @@
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  font-style: normal;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  font-size: bold;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+html {
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
+  color: #3d4e61;
+}
+
+.search-form {
+  margin-top: 20px;
+}
+
+.search {
+  width: 100%;
+  display: block;
+  position: relative;
+  background: #fff;
+}
+
+.search::before {
+  content: "";
+  position: absolute;
+  width: 19px;
+  height: 19px;
+  left: 26px;
+  top: 25px;
+  background: url(images/Search.svg) 50%/contain no-repeat;
+}
+
+.input {
+  width: 100%;
+  line-height: 22px;
+  font-size: 16px;
+  padding: 23px 0 23px 62px;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+.input-small {
+  padding: 10px 0 11px 33px;
+  line-height: 19px;
+  font-weight: 400;
+  font-size: 14px;
+}
+
+.input::placeholder {
+  font-family: Avenir, sans-serif;
+  color: #3d4e61;
+  font-weight: 300;
+  font-size: 16px;
+}
+
+.input:hover {
+  cursor: pointer;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.input:focus {
+  font-weight: 700;
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+  outline: none;
+}
+
+.small::before {
+  content: "";
+  width: 11px;
+  height: 11px;
+  position: absolute;
+  top: 15px;
+  left: 13px;
+  background: url(images/Search.svg) 50%/contain no-repeat;
+}
```

Comment:
You've already added this image to ::before, no need to duplicate it, just override size if you need